lib/services/sign_in.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:google_sign_in/google_sign_in.dart';
+
+class SignIn {
+  static GoogleSignIn googleSignIn = GoogleSignIn(
```

Comment:
keep this under getIt or just in the main file if it doesn't need multi file scope